From 7be703311cec8ec1021a6fd9f10764d5812cd259 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Fri, 14 Sep 2012 09:37:53 -0700
Subject: [PATCH 0524/1357] HDFS-3938. remove current limitations from HttpFS docs. (tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1384843 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 2e895a96156f37fbbe1015a15f55749ffcf73b3f)

Conflicts:

	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt
---
 .../hadoop-hdfs-httpfs/src/site/apt/index.apt.vm   |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/site/apt/index.apt.vm b/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/site/apt/index.apt.vm
index 9a25753..2920cd9 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/site/apt/index.apt.vm
+++ b/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/site/apt/index.apt.vm
@@ -81,8 +81,3 @@ Hadoop HDFS over HTTP - Documentation Sets ${project.version}
 
   * {{{./UsingHttpTools.html}Using HTTP Tools}}
 
-* Current Limitations
-
-  <<<GETDELEGATIONTOKEN, RENEWDELEGATIONTOKEN and CANCELDELEGATIONTOKEN>>>
-  operations are not supported.
-
-- 
1.7.0.4

